modules/installation-guide/examples/system-variables.adoc

```diff
@@ -175,7 +172,7 @@ pass:[<!-- vale off -->]
  `+CHE_WORKSPACE_PLUGIN__REGISTRY__INTERNAL__URL+`,"`+NULL+`","Workspace plugins registry internal endpoint. Should be a valid HTTP URL. Example: ++http://devfile-registry.che.svc.cluster.local:8080++ In case {prod-short} plug-ins registry is not needed value 'NULL' should be used" 
  `+CHE_WORKSPACE_DEVFILE__REGISTRY__URL+`,"`+https://che-devfile-registry.prod-preview.openshift.io/+`","Devfile Registry endpoint. Should be a valid HTTP URL. Example: ++http://che-devfile-registry-eclipse-che.192.168.65.2.nip.io++ In case {prod-short} plug-ins registry is not needed value 'NULL' should be used" 
  `+CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL+`,"`+NULL+`","Devfile Registry 'internal' endpoint. Should be a valid HTTP URL. Example: ++http://plugin-registry.che.svc.cluster.local:8080++ In case {prod-short} plug-ins registry is not needed value 'NULL' should be used" 
- `+CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES+`,"`+persistent,ephemeral,async+`","The configuration property that defines available values for storage types that clients such as the Dashboard should propose to users during workspace creation and update. Available values:   - `persistent`: Persistent Storage slow I/O but persistent.   - `ephemeral`: Ephemeral Storage allows for faster I/O but may have limited storage       and is not persistent.   - `async`: Experimental feature: Asynchronous storage is combination of Ephemeral       and Persistent storage. Allows for faster I/O and keep your changes, will backup on stop       and restore on start workspace.       Will work only if:           - `che.infra.kubernetes.pvc.strategy='common'`           - `che.limits.user.workspaces.run.count=1`           - `che.infra.kubernetes.namespace.allow_user_defined=false`           - `che.infra.kubernetes.namespace.default` contains `<username>`      in other cases remove `async` from the list." 
+ `+CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES+`,"`+persistent,ephemeral,async+`","The configuration property that defines available values for storage types that clients such as the Dashboard should propose to users during workspace creation and update. Available values:   - `persistent`: Persistent Storage slow I/O but persistent.   - `ephemeral`: Ephemeral Storage allows for faster I/O but may have limited storage       and is not persistent.   - `async`: Experimental feature: Asynchronous storage is combination of Ephemeral       and Persistent storage. Allows for faster I/O and keep your changes, will backup on stop       and restore on start workspace.       Will work only if:           - `che.infra.kubernetes.pvc.strategy='common'`           - `che.limits.user.workspaces.run.count=1`           - `che.infra.kubernetes.namespace.default` contains `<username>`      in other cases remove `async` from the list." 
```

Comment:
the output format is strange here, is it intended ?
![image](https://user-images.githubusercontent.com/650571/120295482-73ae1100-c2c7-11eb-92b7-4855a21b6750.png)

I would check with someone from platform team if it is ok for this part.

Reply:
For the time being, any change in this file need to be addressed in the properties files.

We plan to change completely the output format. Tables are barely readable.